lucene/suggest/src/java/org/apache/lucene/search/spell/DirectSpellChecker.java

```diff
@@ -75,6 +75,8 @@
   private float thresholdFrequency = 0f;
   /** minimum length of a query word to return suggestions */
   private int minQueryLength = 4;
+  /** maximum length of a query word to return suggestions */
+  private int maxQueryLength = 0;
```

Comment:
Do we want validation somewhere that max >= min? Or simply treat the max < min case as ignoring max?

Reply:
I prefer to initialize maxQueryLength to Integer.MAX_VALUE, it's more intuitive, so that there is no special <= 0 or < min value.

Reply:
Thanks for the comments - I've put in validation and switched to using Integer.MAX_VALUE.